shared/liboxide/udev.cpp

```
@@ -0,0 +1,261 @@
+    }
+
+    void UDev::subsystem(const QString& subsystem, std::function<void(const Device&)> callback){
+        deviceType(subsystem, NULL, callback);
+    }
+
+    void UDev::subsystem(const QString& subsystem, std::function<void()> callback){
+        deviceType(subsystem, NULL, callback);
+    }
+
+    void UDev::deviceType(const QString& subsystem, const QString& deviceType, std::function<void(const Device&)> callback){
+        connect(singleton(), &UDev::event, [callback, subsystem, deviceType](const Device& device){
```

cppcheck

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)